ql/src/test/results/clientpositive/llap/ppd_deterministic_expr.q.out

```diff
@@ -198,6 +198,9 @@ PREHOOK: query: create view viewDeterministicUDFA partitioned on (vpart1, vpart2
 where part1 in ('US', 'CA')
 PREHOOK: type: CREATEVIEW
 PREHOOK: Input: default@testa
+PREHOOK: Input: default@testa@part1=CA/part2=ABC/part3=300
```

Comment:
This is a CREATE VIEW. Why are partitions that are not accessed part of the entities accessed? This should not have changed? This happens in all these different tests so the root cause should be the same.

Reply:
@jcamachor , I think this behavior is expected. Since in this test, CBO is disabled, ColumnAccessInfo will be null, so on view creation, we'll run step 6-8 after my change. Step 7 is the optimizer, and it runs SimpleFetchOptimizer, which will add these partitions to read from them. Is this answers your concern, or maybe I misunderstood your question?